sdk/nodejs/acmpca/metrics.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as cloudwatch from "../cloudwatch";
+
+export type AcmpcaMetricName =
+    "CRLGenerated" | "MisconfiguredCRLBucket" | "Time" | "Success" | "Failure";
+
+/**
+ * Creates an AWS/ACMPrivateCA metric with the requested [metricName]. See
+ * https://docs.aws.amazon.com/acm-pca/latest/userguide/PcaCloudWatch.html for list of all
+ * metric-names.
+ *
+ * Note, individual metrics can easily be obtained without supplying the name using the other
+ * [metricXXX] functions.
+ */
+export function metric(metricName: AcmpcaMetricName, change: cloudwatch.MetricChange = {}) {

```

Comment:
always have teh general 'metric' helper that all specific metricXXX helpers defer to.  This has the most docs on how the metrics for this package work (some packages go very much in depth).  We should likely have these become README.md info for that package.